com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/LaunchRequestHandler.java

```diff
@@ -145,4 +158,8 @@ public void handle(Command command, Arguments arguments, Response response, IDeb
         }
         sourceProvider.initialize(context.getDebugSession(), options);
     }
+
+    private boolean isEmptyArray(String[] items) {
```

Comment:
Should use since we already refer this lib. 
```
org.apache.commons.lang3.ArrayUtils.isNotEmpty(testArrayName)
```

Reply:
nice